package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "todo-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch",
+    "server": "webpack-dev-server --open",
```

Comment:
**REQUIRED**
Let's change `"server": "webpack-dev-server --open"` to `"start": "webpack-dev-server --open"` since this is the convention used for starting the server.

Reply:
Resolved thank you